### ISSUE TITLE
Can't send Gas from Neon using Ledger Nano S

### ISSUE THREAD

**Issue by jon85943** [CityOfZion/neon-wallet#428]
I get the "Transaction Failed" message every time I try to send Gas on Main using 0.0.7.  I can successfully send Neo, claim my gas, and receive both Neo and Gas.  It's only sending Gas that I cannot get successfully.  I have tried different amounts, decimals, unplugging/replugging in, reopening the Nano app, etc.  I have also tried sending to many different wallets to check if maybe one exchange was incorrect but it still didn't help.  
Also on Test I can send 0 gas and approve the transaction on my ledger Nano, it only fails after I approve because I'm guessing theres nothing to send.

**Comment by stevesbrain** [comment]
I know someone else had this issue, and their solution was using full decimal place amounts. I.e. they were attempting to send `.6613` and it failed, `.6` and it failed, but sending `0.6613` worked (I believe that's what it was anyway). Have you tried sending with the 0.X format?

**Comment by jon85943** [comment]
Thanks for the reply, I saw that post as well and tried .xxxx as well as most other combos.  I also tried using the exact amounts and the 7 digit full number you see when hovering over your Gas amount.  Hopefully v8 fixes it

**Comment by joshzdad2018** [comment]
Having same issue, i can log into a separate neon screen and login with my private key "not the ledger" and send gas from it successfully. but inlog in on the ledger and no gas will send to any address. hmmmmmm tried  every possible scenario.

**Comment by jon85943** [comment]
I had only been storing my NEO on my ledger so I decided to give this a
try: https://coranos.github.io/neo/ledger-nano-s/recovery/

Obviously use it offline but I was able to just enter my Ledger 24 word
phase and it spit out my private key with no issue.  It was the first
option that showed up and was paired with my public key so I knew which one
to select.  I just opened the wallet in Neotracker.io and moved it into the
new wallet I had created.

**Comment by AlphaCVK** [comment]
Same issue, can claim gas with ledger but unable to send to any address...

**Comment by joshzdad2018** [comment]
ended up having to add a few zeroes to the end of the amount then it went
thru. don't remember exactly how many will try again later when gas
rejuvenates.

On 3 Jan 2018 4:56 pm, "AlphaCVK" <notifications@github.com> wrote:

Same issue, can claim gas with ledger but unable to send to any address...

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/CityOfZion/neon-wallet/issues/428#issuecomment-355170690>,
or mute the thread

**Comment by AlphaCVK** [comment]
Yup, it works now, you have to enter the amount followed point 7 numerals, i.e. for 1 gas it would be 1.0000000

**Comment by joshzdad2018** [comment]
Yup, right on thanks for the info! have s great one! hioe your NEO
Skyrockets!

**Comment by tomocean22** [comment]
I still can't send gas. Is there a minimum to send? Has anyone had luck sending amounts around .10, what decimal format did you use? 

I've tried reinstalling both NEON and the NEO ledger app. I get the same error as the OP where the transaction fails with no prompt appearing on my ledger.

**Comment by joshzdad2018** [comment]
Try installing the new version of neon. If you havent already.

https://github.com/CityOfZion/neon-wallet/releases

**Comment by tomocean22** [comment]
The problem persists for me with version 0.0.9

**Comment by Thieazs** [comment]
Same problem for me with Version 0.0.9 and Ledger Nano S - Cant claim GAS either as the onion to sign the transaction does not appear on the Ledger - please help

**Comment by joshzdad2018** [comment]
Hmmmmmmm,  I have windows 10, uninstalled old version of neon wallet, installed new version 9, uninstalled my neo app on ledger S, installed it again onto the ledger then it started sending gas any amount.

**Comment by cryptoarmeen** [comment]
can someone help me please i cant send gas from neo wallet 0.0.9

**Comment by stevesbrain** [comment]
@cryptoarmeen Tried 0.1.0?

**Comment by cryptoarmeen** [comment]
I will try thank you!

Verstuurd vanaf mijn iPhone

Op 18 jan. 2018 om 11:18 heeft Steve Divskinsy <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


@cryptoarmeen<https://github.com/cryptoarmeen> Tried 0.1.0?

**Comment by Salzburger1970m** [comment]
Hello - I just have the same problem. I cant sed GAS out of my NEO Wallet 
Version 0.1.1 connected via Ledger Nano S OS Win10Pro64. 
But funny is that i can send the amount of 0.00000016 GAS or 0.00000001 - other amounts i tried dont work! Please help :-)

**Comment by tomocean22** [comment]
For me, version 0.1.1 fixes the problem. I can now send GAS with the ledger nano

**Comment by chenshak** [comment]
I still have this problem (version 0.1.1).
Tried to send 0.00000001 GAS using my ledger many times and every time got 'transaction failed' message (don't got a message on the ledger to sign transaction).
Also, when clicking the Max button it fills 18 instead of 0.00000001 (although when clicking the Next button then on the next screen it shows the correct value 0.00000001).

**Comment by ToeJamathon** [comment]
Likewise having this same problem :(

Tried with 0.1.1, 0.1.0 and 0.0.9 (I'd like to try with 0.0.7, but I can't seem to get the downloadable installer from: https://github.com/CityOfZion/neon-wallet/releases - if someone could help me find the installer for 0.0.7 that'd be great!)

Anyway, I've tested with the 3 versions above so far, and of course tried removing/reinstalling the NEO app from my Ledger multiple times.

@jon85943 I'd really like to try your solution, but because I've had multiple coins on my ledger, when I put my private key into https://coranos.github.io/neo/ledger-nano-s/recovery/ (offline of course) I couldn't figure out which public key/private key pair was my GAS (or NEO for that matter...). Do you know how would be the best way to get my NEO or GAS private keys the site...?

**Comment by joeman205** [comment]
This is such a problem. I am glad to see that other people are having it also. 
I can't sent ANY gas from my ledger nano s to anywhere (specifically Binanace). 
The new version of NEON wallet did NOT fix this. I have tried EVERY different way to type in a number to send.... PLEASE HELP!   
Joe

**Comment by stevesbrain** [comment]
@joeman205 @cryptoarmeen Do you guys have the same issue with testnet GAS?

**Comment by cryptoarmeen** [comment]
Yes same isue but with mainet GAS

Verstuurd vanaf mijn iPhone

Op 23 jan. 2018 om 01:57 heeft Steve Divskinsy <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


@joeman205<https://github.com/joeman205> @cryptoarmeen<https://github.com/cryptoarmeen> Do you guys have the same issue with testnet GAS?

**Comment by stevesbrain** [comment]
@cryptoarmeen I mean, is this *only* an issue with mainnet, or does it also happen on testnet? :)

**Comment by cryptoarmeen** [comment]
Mine GAS is only on mainnet i dont have gas on testnet so i cant try :(

Verstuurd vanaf mijn iPhone

Op 24 jan. 2018 om 13:06 heeft Steve Divskinsy <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


@cryptoarmeen<https://github.com/cryptoarmeen> I mean, is this only an issue with mainnet, or does it also happen on testnet? :)

**Comment by stevesbrain** [comment]
@cryptoarmeen What is your address? I'll send you some testnet gas so you can try :)

**Comment by joeman205** [comment]
I've only used mainnet on NEON, my balances do not show up on testnet. I am still having the same issue though.

**Comment by cryptoarmeen** [comment]
please help me send gas from my neo wallet I use the latest 0.1.2 wallet but I still can not send gas. I always get the message transaction failed! i love NEO but they still can not fix this problem! I have all my NEO and GAS on my ledger nano s please help me!

**Comment by cryptoarmeen** [comment]
@stevesbrain 
I have my NEO and GAS on mainnet testnet is empty

**Comment by stevesbrain** [comment]
@cryptoarmeen 0.0.7 has been removed from available downloads unfortunately.

**Comment by Laurentius26** [comment]
Hi all, same issue. I use Neon desktop wallet v.0.1.2 with Ledger Nano S but GAS is stuck in the wallet. I can't send it, always getting 'transaction failed'. 10 GAS I have there, a lot of money for me. :-(

**Comment by bobzilla1007** [comment]
Hello everyone. I was having this issue with my ledger nano S. I could not SEND gas from it. I would get the red transaction error (couldn't sign)
I have found a way to get it working and I will share my experience hopefully it helps others. 

The first step which I'm not sure was necessary but in the interest of accuracy I will list it. 
1)Changed  block explorer in "manage neon settings" from NeoTracker to Neo Scan. 
2)Tried sending 0.00000100 GAS to an address and it went through!
3)Changed block explorer back to NeoTracker
4)Sent again 0.00000100 and it worked
5)Tried sending 1 GAS failed
6)Sent 0.00000500 and it worked
7)Tried various amounts and it failed
8)Sent 0.01000000 and it worked
9)Sent 0.90000000 and it worked
10) Sent 1.00000000 GAS it worked

I'm not sure here but it seemed that I had to start with a very small amount and as I worked my way up it started working. Perhaps a memory allocation problem, just don't know. 

If anyone else tries this method and it works please comment back and spread the word for others. I do not have much social activity online so I cannot help in that. I opened an account here just to post this message becasue I thought it important enough.

**Comment by cryptoarmeen** [comment]
It worked thank you soo much friend!!

Verstuurd vanaf mijn iPhone

Op 11 feb. 2018 om 17:33 heeft bobzilla1007 <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


Hello everyone. I was having this issue with my ledger nano S. I could not SEND gas from it. I would get the red transaction error (couldn't sign)
I have found a way to get it working and I will share my experience hopefully it helps others.

The first step which I'm not sure was necessary but in the interest of accuracy I will list it.
1)Changed block explorer in "manage neon settings" from NeoTracker to Neo Scan.
2)Tried sending 0.00000100 GAS to an address and it went through!
3)Changed block explorer back to NeoTracker
4)Sent again 0.00000100 and it worked
5)Tried sending 1 GAS failed
6)Sent 0.00000500 and it worked
7)Tried various amounts and it failed
8)Sent 0.01000000 and it worked
9)Sent 0.90000000 and it worked
10) Sent 1.00000000 GAS it worked

I'm not sure here but it seemed that I had to start with a very small amount and as I worked my way up it started working. Perhaps a memory allocation problem, just don't know.

If anyone else tries this method and it works please comment back and spread the word for others. I do not have much social activity online so I cannot help in that. I opened an account here just to post this message becasue I thought it important enough.

**Comment by Laurentius26** [comment]
@bobzilla1007 , confirmed working, A bit of hazzle but it works. Thank you very much, thnak you all. :-)

**Comment by bobzilla1007** [comment]
I'm glad it has helped. 

Side note:
Since I posted this I have watched the price of gas start going sideways and down. Of course it couldn't be proven but I think everyone with this problem is now flooding the market and selling cheap.

Please people don't run out and sell off your gas cheap and out bid each other just because your excited to have access to your coins.  Now I wish I sold before I posted ;-(

**Comment by cryptoarmeen** [comment]
Im not selling just want to move to binance !

Verstuurd vanaf mijn iPhone

Op 11 feb. 2018 om 20:38 heeft bobzilla1007 <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


I'm glad it has helped.

Side note:
Since I posted this I have watched the price of gas start going sideways and down. Of course it couldn't be proven but I think everyone with this problem is now flooding the market and selling cheap.

Please people don't run out and sell off your gas cheap and out bid each other just because your excited to have access to your coins. Now I wish I sold before I posted ;-(

**Comment by stefanxg** [comment]
Problem:  I transferred 5 GAS to my (legder nana S) Neo wallet adress from Poloniex.
Nothing showed up ?  Transaction on Poloniex is completed normally.
Can somebody please give an advise ?  Thanx in advance. 
Note: I  I transferred 7 Neo to my (legder nana S) Neo wallet adress from Bittrex, all oke!

**Comment by tyranosa** [comment]
Thanks  bobzilla1007 . It work for me. Now i can send 0.29xxxxxx GAS.

**Comment by alatalot** [comment]
This is really a major problem! I cannot send any tokens (DBC, QLC) or claim GAS with newest NEON and Ledger Nano S as it doesn't let me sign it just showing red transaction failed. There's only one way to fix this and it's not any trick n.o 6 start working your way up. I hope this is solved quickly!

**Comment by bobzilla1007** [comment]
Hello everyone, 
I downloaded the latest NEON 0.1.3 recently and tried sending some gas. I was hoping something in the new release would have a fix to this bug but it still behaved the same for me on the first try. 

I am not saying anything new here but in case the devs are reading it's just a feedback from a user's experience. 

I shortened up my work around this time because I didn't think that first step was necessary and it appeared not to be. 

I did a transfer of 0.00000100 and it worked
I then just went right to sending one 1.00000000 and it worked. 

I think their are two things of importance to observe. First send that small amount first which seems to get the sighing pop up to appear. The second thing seems to be that you want to always use eight decimal places, (fill with zeros to the eighth position)

Perhaps also using simple fractions such as 0.00100000  and not something like 0.00102497 may be a factor too, I tend to use clean numbers like that when I send.

BTW:
Apologizes for my whining about the gas price previously ;-) It's not for me to tell anyone what to do with their funds.
